requirements.txt

```
@@ -1,5 +1,6 @@
 Babel==2.9.1  # min version = 2.6.0 (Focal with security backports)
 chardet==3.0.4
+cryptography<37.0.0  # remove this when pyopenssl>=21.0.0
```

```suggestion
cryptography<=36.0.2  # remove this when pyopenssl>=21.0.0
```
This can be an alternative, but I don't know if it's better this way or not. As you prefer.

@odony